mutex_test.go

```diff
@@ -284,6 +287,27 @@ func (s *mutexSuite) TestMutexNotInherited(c *gc.C) {
 	r.Release()
 }
 
+// TestFilePermissions tests that the file permissions are correct.
+func (s *mutexSuite) TestFilePermissions(c *gc.C) {
```

Comment:
While kinda useful, this test doesn't actually test the code you have written at all because it will never be run with sudo.

Reply:
Yeah, I'll make a complimentary test function, good spot.

Reply:
fixed in https://github.com/juju/mutex/pull/6/commits/b5c51c193415eabe150a62058ec2985c57395433